core/src/test/scala/org/apache/spark/status/api/v1/SimpleDateParamTest.scala

```diff
@@ -21,6 +21,11 @@ import org.scalatest.{Matchers, FunSuite}
 class SimpleDateParamTest extends FunSuite with Matchers {
 
   test("date parsing") {
+    import java.util.{Locale, TimeZone}
```

Comment:
Nit: why import these so locally?
I'm slightly afraid this will have negative effects on other tests since you're setting the defaults globally for the JVM.
Why would this fail though if the date parameter specifies its time zone? it should not be ambiguous.